### ISSUE TITLE
Museum: Clean up iMac storage on the floor

### ISSUE THREAD

**Issue by wynnmade** [Museum-of-Art-and-Digital-Entertainment/MADE-Initiatives#16]
iMac is running out of space and needs someone to look at why! It would also be great to define some sort of administration practice to reduce these occurrences.

**Comment by wynnmade** [comment]
Migrate the machine to an SSD/reformat, image, etc.

**Comment by andrewwatterson** [comment]
I'm happy to come take a look at this.

@qst0 or @asajbel are there better/worse times to do this? I'm pretty flexible and don't want to take the iMac out of service at peak times.

Imaging Classic Mac OS disks isn't that complicated — opinions seem split on whether you can just use Disk Copy or need something heftier like [Carbon Copy Cloner](https://macintoshgarden.org/apps/carbon-copy-cloner-347)

What OS is on it right now (I've definitely got OS 9.x install media, but I could rustle up 8.6 if necessary)? I'll come prepared with install media, a flash drive for booting/backups, and any tools I might need.

**Comment by vonguard** [comment]
I went and deleted all the archives and disk images that were on the hard drive and we're in the games folder. That freed up 1.2 gigs. Everyone thinks
Opens the folders. Mouse sucks. Having a better mouse fix this problem but I think we need to do a fresh reinstall of 9.2 because a lot of stuff has been removed from the menus. There are free 9.2 images out. There can be burned onto a CD and reinstalled. But then we still have to reinstall the games.
